CHANGELOG.md

```diff
@@ -1,4 +1,19 @@
 ## 1.10.0 (Unreleased)
+
+FEATURES
+
+* provider/scaleway: update terraform SDK to 0.12.0 [GH-127]
+
+IMPROVEMENTS
+
+* provider/scaleway: add .vscode to .gitignore [GH-128]
```

Comment:
nit: we usually only include customer facing improvements.

Reply:
Thanks for the feedback, I will fix it